libs/easyloggingpp/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=easyloggingpp
+PKG_VERSION:=9.97.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/abumq/easyloggingpp.git
+PKG_SOURCE_VERSION:=v9.97.0
+PKG_SOURCE_DATE:=2020-12-25
+
+PKG_MIRROR_HASH:=9acdb77ca418b07feef4ffc4ae1858c0a11f9e44c44a0eff21d3e278844e4dc1
```

Comment:
use codeload instead, for example:
https://github.com/openwrt/packages/blob/1cb2590c1743eeb4c357b1f0d7e3fb47b3640ae6/utils/yq/Makefile#L7-L9